src/UglyToad.PdfPig/Export/AltoXmlTextExporter.cs

```diff
@@ -0,0 +1,5337 @@
+        /// <param name="wordExtractor"></param>
+        /// <param name="pageSegmenter"></param>
+        /// <param name="scale"></param>
+        /// <param name="indent">Indent character.</param>
+        public AltoXmlTextExporter(IWordExtractor wordExtractor, IPageSegmenter pageSegmenter, double scale = 1.0, string indent = "\t")
+        {
+            this.wordExtractor = wordExtractor;
+            this.pageSegmenter = pageSegmenter;
+            this.scale = (decimal)scale;
+            this.indentChar = indent;
+        }
+
+        /// <summary>
+        /// Get the Alto (XML) string of the pages layout.
+        /// </summary>
+        /// <param name="document"></param>
+        /// <param name="includePaths">Draw <see cref="PdfPath"/>s present in the page.</param>
+        /// <returns></returns>
+        public string Get(PdfDocument document, bool includePaths = false)
+        {
+            AltoDocument alto = CreateAltoDocument("unknown");
+            var altoPages = new List<AltoDocument.AltoPage>();
+
+            for (var i = 0; i < document.NumberOfPages; i++)
+            {
+                var page = document.GetPage(i + 1);
+                altoPages.Add(ToAltoPage(page, includePaths));
+            }
+            alto.Layout.Pages = altoPages.ToArray();
+
+            return Serialize(alto);
+        }
+
+        /// <summary>
+        /// Get the Alto (XML) string of the page layout. Excludes <see cref="PdfPath"/>s.
+        /// </summary>
+        /// <param name="page"></param>
+        /// <returns></returns>
+        public string Get(Page page)
+        {
+            return Get(page, false); 
+        }
+
+        /// <summary>
+        /// Get the Alto (XML) string of the page layout.
+        /// </summary>
+        /// <param name="page"></param>
+        /// <param name="includePaths">Draw <see cref="PdfPath"/>s present in the page.</param>
+        /// <returns></returns>
+        public string Get(Page page, bool includePaths)
+        {
+            AltoDocument alto = CreateAltoDocument("unknown");
+            var altoPages = new List<AltoDocument.AltoPage>();
+
+            alto.Layout.Pages = new AltoDocument.AltoPage[] { ToAltoPage(page, includePaths) };
+
+            return Serialize(alto);
+        }
+
+        /// <summary>
+        /// Create an empty <see cref="AltoDocument"/>.
+        /// </summary>
+        /// <param name="fileName"></param>
+        /// <returns></returns>
+        private AltoDocument CreateAltoDocument(string fileName)
+        {
+            return new AltoDocument()
+            {
+                Layout = new AltoDocument.AltoLayout()
+                {
+                    StyleRefs = null
+                },
+                Description = GetAltoDescription(fileName),
+                SchemaVersion = "4",
+                //Styles = new AltoStyles() { },
+                //Tags = new AltoTags() { }
+            };
+        }
+
+        /// <summary>
```

Comment:
I'd remove these empty doc comments, since they're not needed on private or internal members.